insuree/schema.py

```diff
@@ -309,15 +321,15 @@ def on_family_and_insuree_mutation(kwargs):
 
 def on_mutation(sender, **kwargs):
     return {
-        CreateFamilyMutation._mutation_class: lambda x: on_family_mutation(x),
-        UpdateFamilyMutation._mutation_class: lambda x: on_family_mutation(x),
-        DeleteFamiliesMutation._mutation_class: lambda x: on_families_mutation(x),
-        CreateInsureeMutation._mutation_class: lambda x: on_insurees_mutation(x),
-        UpdateInsureeMutation._mutation_class: lambda x: on_insurees_mutation(x),
-        DeleteInsureesMutation._mutation_class: lambda x: on_family_and_insurees_mutation(x),
-        RemoveInsureesMutation._mutation_class: lambda x: on_family_and_insurees_mutation(x),
-        SetFamilyHeadMutation._mutation_class: lambda x: on_family_mutation(x),
-        ChangeInsureeFamilyMutation._mutation_class: lambda x: on_family_and_insuree_mutation(x),
+        CreateFamilyMutation._mutation_class: on_family_mutation,
+        UpdateFamilyMutation._mutation_class: on_family_mutation,
+        DeleteFamiliesMutation._mutation_class: on_families_mutation,
+        CreateInsureeMutation._mutation_class: on_insurees_mutation,
+        UpdateInsureeMutation._mutation_class: on_insurees_mutation,
+        DeleteInsureesMutation._mutation_class: on_family_and_insurees_mutation,
+        RemoveInsureesMutation._mutation_class: on_family_and_insurees_mutation,
+        SetFamilyHeadMutation._mutation_class: on_family_mutation,
+        ChangeInsureeFamilyMutation._mutation_class: on_family_and_insuree_mutation,
```

Comment:
Just a small more idiomatic Python. No need for a lambda to just pass a function reference.